menuinst/platforms/osx.py

```diff
@@ -74,6 +74,14 @@ def _precreate(self):
             os.symlink(self.render(src), rendered_dest)
 
     def create(self) -> Tuple[Path]:
+        if self.location.exists():
+            message = (
+                f"App already exista at {self.location}. "
+                "Please remove the existing shortcut before installing. "
+                "If you used conda to install this package, "
+                "re-run the installation with --no-shortcuts."
```

Comment:
I think conda will not raise this exception, and just warn about it. So the message could be instead something like:

```suggestion
                "If you used conda to install this package, "
                "reinstall the package with --force-reinstall to "
                "create the shortcut once the location is cleared."
```